### ISSUE TITLE
A11y_Windows Template Studio_Select a Project type_Screenreader: Screenreader is not announcing Information, which is after Navigation Pane.

### ISSUE THREAD

**Issue by Yash14j** [microsoft/WindowsTemplateStudio#3892]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) – Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yTCS;#GH_WindowsTemplateStudio_Oct2020;#FTP;#STP;#NVDA;#Narrator;#A11yMAS;#A11ySev2;#Windows Template Studio;#DesktopApp;#Win10;#WCAG1.3.1;#NonBenchmark;

### **Environment Details:**
Application Name: Windows Template Studio (WinTS)
Templates version: 3.8.20273.1
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.8.0 Preview 3.1
Windows Version: Windows10
NVDA Version 2019.3.1

### **Repro Steps:**

1. Open Visual Studio 2019 
2. Navigate to File from menu with the help of tab key and press enter key to expand it. 
3. Select new, press enter to expand and then select project.
4. Select the "Windows template studio (WPF.Net Core)". Navigate to next button with the help of tab key and press enter key
5. "Configure your new project" screen will appear. 
6. Navigate to create button with the help of tab key and press enter key.
7. New WPF App screen will open.
8. Open screenreader and navigate to “Navigation Pane” with the help of tab key.
9. Now with the help of “Caps+ down arrow” key try to access information.

### **Actual Result:**
Screenreader is not announcing the content after Navigation pane heading in list item “Navigation Pane” with the help of “Caps+ down arrow” or “Caps+ right arrow” key.

### **Note:**
This issue is observed with both NVDA and NARRATOR.
This issue is observedin following scenarios:**
•	Select a Design pattern
•	Add Pages
•	Add Features
•	Select The Template You Want To Add
•	Add Services

### **Expected Result:**
Screenreader should announce the content after Navigation pane heading in list item “Navigation Pane” with the help of “Caps+ down arrow” or “Caps+ right arrow” key.

### **User Impact:**
Screenreader user will not get proper and complete information about “Navigation Pane”.

**Comment by sibille** [comment]
Verified in dev-nightly version:
Wizard version: 0.22.20310.1
You can install the dev-nightly version from our nightly feed (https://www.myget.org/F/windows-template-studio-nightly/vsix/).
For instructions on how install WTS from the dev-nightly feed in Visual Studio see https://github.com/microsoft/WindowsTemplateStudio/blob/dev/docs/getting-started-extension.md#nightly--pre-release-instructions

**Comment by Yash14j** [comment]
Issue does not repro in the Wizard version: 0.22.20310.1 hence closing this bug.
